### PR TITLE
Added metadata field to lti tool model, visible to admin only (#10)

### LINKED ISSUE #10: Have a place for admins to store internal metadata (like notes and tool configuration information)

### PR DESCRIPTION
**Pull request by roujiawang**
Resolved #10

### CODE REVIEW

File: backend/canvas_app_explorer/models.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ class LtiTool(models.Model):
     privacy_agreement = HTMLField()
     support_resources = HTMLField()
     canvas_placement = models.ManyToManyField(CanvasPlacement)
+    metadata = HTMLField(null=True)
```
Comment (ssciolla) [inline backend/canvas_app_explorer/models.py:42]: @jonespm, how do we feel about the name `metadata`, as opposed to just `notes` or `internal_notes`?

Comment (ssciolla) [reply backend/canvas_app_explorer/models.py:42]: I could see adding `help_text` here to indicate this isn't shown to users, especially if we don't change the name to internal. Something like this:
```suggestion
    metadata = HTMLField(null=True, help_text='Field for internal notes (not shown to users)')
```

**Review by ssciolla — COMMENTED** [review]
This is a functional and seems like it would be useful, at least during the content creation or editing phase for pasting snippets or tracking progress on review, etc. I just wonder if `metadata` is the appropriate name. I'd like to hear @jonespm's or @ziegler1960's opinion.

File: backend/canvas_app_explorer/models.py (hunk 1 above)
Comment (jonespm) [reply backend/canvas_app_explorer/models.py:42]: Yeah, I think internal_notes sounds like a better field name.

Comment (ziegler1960) [reply backend/canvas_app_explorer/models.py:42]: Reading thru this request it's not entirely clear to me why we need this field and what will be entered.  Nonetheless, I agree that Internal notes is better than 'metadata'

**Review by ssciolla — APPROVED** [review]
Nice work! With one approval, you can merge this now. Click "Squash and merge" at the bottom when you're ready.

**Pull request merged**